test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -178,6 +274,11 @@ wt_standalone=false
 newer_release_branches=(develop mongodb-5.0 mongodb-4.4 mongodb-4.2)
 older_release_branches=(mongodb-4.2 mongodb-4.0 mongodb-3.6)
 
+# This array is used to generate compatible configuration files between releases, because
+# upgrade/downgrade test runs each build's format test program on the second build's
+# configuration file. 
+compatible_upgrade_downgrade_releases=(mongodb-4.4 mongodb-4.2)
```

Comment:
Why there are only 2 branches included while the array name and comment are general? 
Do we need to have the compatible config generated for other release branches? 
Suggest to add "branches" into the array name (as they are technically release branches vs releases :))

Reply:
As discussed the upgrade/downgrade testing is done on the last two release branches from `newer_release_branches`. So, this array should contain the release branches that understand each other's config files. Updated the array name.